packages/core/strapi/lib/core/loaders/load-middlewares.js

```diff
@@ -4,14 +4,15 @@
 const path = require('path');
 const fs = require('fs-extra');
 const _ = require('lodash');
-const glob = require('../load/glob');
-const findPackagePath = require('../load/package-path');
+const glob = require('../../load/glob');
```

Comment:
TODO after merging: We will need to use the middleware registry instead so we put everything into it and use it in the initializeMiddlewares later on